frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -54,7 +55,9 @@ const TranslationPage = () => {
   const [changedStoryLines, setChangedStoryLines] = useState<
     Map<number, StoryLine>
   >(new Map());
-  const [percentageComplete] = useState(25);
+  const [numTranslatedLines, setNumTranslatedLines] = useState(0);
+  const [numStoryLines, setNumStoryLines] = useState(0);
```

Comment:
Is `numStoryLines` necessary? Is it ever different from the length of `translatedStoryLines`, would it ever need to trigger state change?

Reply:
nope will remove that!